src/Book.js

```diff
@@ -0,0 +1,58 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {sendJson} from './sendJson';
+
+export const Book = ({id}) => {
+  const [bookState, setBookState] = useState([]);
```

Comment:
The default should probably be undefined if it hasn't been loaded yet. So `useState()` instead of `useState([])`